api/commands/format.ts

```diff
@@ -1,288 +1,6 @@
-    !formattedPlaceholder.includes("\n")
-  ) {
-    return sqlx.replace(placeholderId, formattedPlaceholder.trim());
-  }
-  // Push multi-line placeholders to their own lines, if they're not already on one.
-  const [textBeforePlaceholder, textAfterPlaceholder] = wholeLine.split(placeholderId);
-  const newLines: string[] = [];
-  if (textBeforePlaceholder.trim().length > 0) {
-    newLines.push(`${indent}${textBeforePlaceholder.trim()}`);
-  }
-  newLines.push(formattedPlaceholder);
-  if (textAfterPlaceholder.trim().length > 0) {
-    newLines.push(`${indent}${textAfterPlaceholder.trim()}`);
-  }
-  return sqlx.replace(wholeLine, newLines.join("\n"));
-}
-
-function formatSqlQueryPlaceholder(node: SyntaxTreeNode, jsIndent: string): string {
-  switch (node.type) {
-    case SyntaxTreeNodeType.JAVASCRIPT_TEMPLATE_STRING_PLACEHOLDER:
-      return formatJavaScriptPlaceholder(node, jsIndent);
-    case SyntaxTreeNodeType.SQL_LITERAL_STRING:
-    case SyntaxTreeNodeType.SQL_COMMENT:
-      return formatEveryLine(node.concatenate(), line => `${jsIndent}${line.trimLeft()}`);
-    default:
-      throw new Error(`Unrecognized SyntaxTreeNodeType: ${node.type}`);
-  }
-}
-
-function formatJavaScriptPlaceholder(node: SyntaxTreeNode, jsIndent: string) {
-  const formattedJs = formatJavaScript(node.concatenate());
-  const textInsideBraces = formattedJs.slice(
-    formattedJs.indexOf("{") + 1,
-    formattedJs.lastIndexOf("}")
-  );
-  // If the formatted JS is only a single line, trim all whitespace so that it stays a single line.
-  const finalJs = textInsideBraces.trim().includes("\n")
-    ? `\${${textInsideBraces}}`
-    : `\${${textInsideBraces.trim()}}`;
-  return formatEveryLine(finalJs, line => `${jsIndent}${line}`);
-}
-
-function formatEveryLine(text: string, mapFn: (line: string) => string) {
-  return text
-    .split("\n")
-    .map(mapFn)
-    .join("\n");
-}
-
-function getWholeLineContainingPlaceholderId(placeholderId: string, text: string) {
-  const regexpEscapedPlaceholderId = placeholderId.replace(/[.*+?^${}()|[\]\\]/g, "\\$&");
-  // This RegExp is safe because we only use a 'placeholderId' that this file has generated.
-  // tslint:disable-next-line: tsr-detect-non-literal-regexp
-  return text.match(new RegExp(".*" + regexpEscapedPlaceholderId + ".*"))[0];
-}
-
-function postProcessFormattedSqlx(formattedSql: string) {
-  let previousLineHadContent = false;
-  formattedSql = formattedSql.split("\n").reduce((accumulatedSql, currentLine) => {
-    const lineHasContent = currentLine.trim().length > 0;
-    if (lineHasContent) {
-      previousLineHadContent = true;
-      return `${accumulatedSql}\n${currentLine.trimRight()}`;
-    }
-    if (previousLineHadContent) {
-      previousLineHadContent = false;
-      return `${accumulatedSql}\n`;
-    }
-    return accumulatedSql;
-  }, "");
-  return `${formattedSql.trim()}\n`;
-}
+/**
+ * @deprecated Use formatter in @dataform/sqlx.
+ */
+export { format, formatFile };
```

Comment:
IMO it's fine to delete this now, it's a very small fix in the open->closed source merge to point at the new location.

Reply:
SGTM